pkgs/development/compilers/gambit/unstable.nix

```diff
@@ -1,12 +1,13 @@
-{ stdenv, callPackage, fetchgit }:
+{ stdenv, callPackage, fetchFromGitHub }:
 
 callPackage ./build.nix {
-  version = "unstable-2018-09-03";
-# git-version = "4.9.0";
-  SRC = fetchgit {
-    url = "https://github.com/feeley/gambit.git";
-    rev = "7cdc7e7b9194b2c088c0667efaf7686a4ffd0d8a";
-    sha256 = "06mmi8jkinihfirz4gjfw2lhxhskiqf3d47sihzx10r60asyqcxg";
+  version = "unstable-2018-11-16";
+# git-version = "4.9.1";
+  SRC = fetchFromGitHub {
+    owner = "feeley";
+    repo = "gambit";
+    rev = "79f5bf774e89c1220bece667598877821ca94b7c";
```

Comment:
```suggestion
    version = "4.9.1";
    rev = "v${version}";
```

Reply:
This can be also become part of the package name instead of having a date.

Reply:
Uh? I was told that for unstable, the date was the standard convention for the package version.

Also, as modifications are made after release, the unstable will track the master branch.

Reply:
Yes, we do this if we don't have a version number.
However this is not the case here, right?

Reply:
It's generated automatically by an update script that tracks the git branch `master`. I just re-ran it and already now there is no simple version number usable the way you suggest.